cleanup_pods/command_line.py

```diff
@@ -0,0 +1,32 @@
+import argparse
+from kubernetes import client, config
+
+
+def delete_pod(name, namespace):
+    core_v1 = client.CoreV1Api()
+    delete_options = client.V1DeleteOptions()
+    api_response = core_v1.delete_namespaced_pod(
+        name=name,
+        namespace=namespace,
+        body=delete_options)
+    print(api_response)
+
+
+def cleanup(args):
+    config.load_kube_config()
+    core_v1 = client.CoreV1Api()
+    pod_list = core_v1.list_namespaced_pod(args.namespace)
```

Comment:
Can we do this instead?

```suggestion
    pod_list = core_v1.list_namespaced_pod(args.namespace, field_selector=''status.reason==Evicted'')
```

Reply:
@ashb what about `failed` and `succeeded`
>Cleanup pods in evicted/failed/succeeded state #508
?

Reply:
@ashb what do you think?

Reply:
Oh yeah I forgot failed/succeeded too.